app/controllers/stripe_active_cards_controller.rb

```diff
@@ -11,16 +11,14 @@ def create
       Rails.logger.info("Stripe Add New Card Success - #{current_user.username}")
       flash[:settings_notice] = "Your billing information has been updated"
     else
-      DatadogStatsClient.increment("stripe.errors")
+      DatadogStatsClient.increment("stripe.errors", tags: ["action:create", "user_id:#{current_user.id}"])
```

Comment:
I think the tag should be `action:create_card` or `action:create:card`, as Stripe can create many things